.github/workflows/publish.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - name: Wait for tests
+        uses: lucidsoftware/wait-on-check-action@v0.1-opt-auth
```

Comment:
this blocks until master's green; [more info](https://github.com/marketplace/actions/wait-on-check) (currently using a fork cause we don't need the auth token for public repos; [PR to upstream](https://github.com/lewagon/wait-on-check-action/pull/8))